couchdb/couchdb.rb

```diff
@@ -81,7 +81,7 @@ def parent_interrupt
   ['--apihost',   '-a', GetoptLong::REQUIRED_ARGUMENT]
```

Comment:
run any of the ruby code quality tools to verify code style warning and error. resolve if possible